### PR TITLE
Update flywheel.md

### PR DESCRIPTION
**Pull request by lauren-stein**
Can we also change the file name to growthloop.md

# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by lauren-stein** [comment]
Hi Isaiah,

Thanks for letting me know. Would we be able to update our Partner Page on
Braze's website without signing the agreement?

This allows Braze to license to use and reproduce any code we deliver.

Best,
Lauren

**Comment by internetisaiah** [comment]
Hi @lauren-stein,

Hope you're doing well today! Thanks for sharing your concerns!

Braze cannot provide legal advice to any parties outside of its the company. We encourage you to speak to your relevant legal counsel, should you have any questions about the CLA.

Please note that all contributors are required to sign the CLA before contributing and that the original submitter of the FlyWheel partner docs has already signed this CLA.

Feel free to reach out with any follow-up questions. Once you sign the agreement, we'll be able to get your changes merged shortly!

Best,
Isaiah

**Review by lauren-stein — COMMENTED** [review]
@cla-bot check

**Comment by internetisaiah** [comment]
@lauren-stein Awesome! Thanks for getting that done!

I've already started working on my part of updates and my goal is to get them done before the weekend. I'll reach out if I run into any blockers that would need your attention, otherwise I'll tag you here once my changes have been added.

Have a great rest of your day!

**Comment by lauren-stein** [comment]
Awesome, thank you!

**Comment by internetisaiah** [comment]
Hey there @lauren-stein ,

Hope your Friday's wrapping up well. Just wanted to let you know that I updated the file name and mentions of "Flywheel" throughout the docs site. Currently working on getting the tile icon/name on these pages updated next (which is the last step):

- [Braze Docs: Partners Homepage](https://www.braze.com/docs/partners/home)
- [Braze Docs: Data & Infrastructure Agility Homepage](https://www.braze.com/docs/partners/data_and_infrastructure_agility)

Once I'm done with that, this update should be complete. Is there any other changes you were hopping to make at this time?

**Comment by lauren-stein** [comment]
Hi Isaiah,

That all sounds great. I looked at the page tonight and it still said
Flywheel everywhere, so please let me know when it is all updated!

Best,
Lauren

**Comment by lauren-stein** [comment]
Hello,

Is there any update on this? I checked the page today and it still says
Flywheel.

Thanks,
Lauren

**Comment by internetisaiah** [comment]
Hello @lauren-stein ,

Apologies for the delay here! It looks like we are still working with your team to get an image we can add to the tiles on the following pages:

- [Braze Docs: Partners Homepage](https://www.braze.com/docs/partners/home)
- [Braze Docs: Data & Infrastructure Agility Homepage](https://www.braze.com/docs/partners/data_and_infrastructure_agility)

Once that's done, I'll be able to make the final tweaks to this PR then get all the changes merged at once! For context, we typically deploy our changes in the `develop` branch to the live docs site every Tuesday and Thursday.

File: assets/js/broken_redirect_list.js [hunk 1]
```diff
@@ -1073,3 +1073,5 @@ validurls['/docs/hidden/private_betas/tiktok'] = '/docs/partners/canvas_steps/ti
 validurls['/docs/email_engagement_dashboard/'] = '/docs/user_guide/data_and_analytics/analytics/email_performance_dashboard/#email-insights-dashboard/';
 
 validurls['/docs/user_guide/administrative/app_settings/email_settings/#include-a-list-unsubscribe-header'] = '/docs/user_guide/administrative/app_settings/email_settings/#enable-list-unsubscribe-header';
+
+validurls['/docs/partners/data_and_infrastructure_agility/workflow_automation/flywheel'] = '/docs/partners/data_and_infrastructure_agility/workflow_automation/growthloop';
```
Comment (bre-fitzgerald) [inline assets/js/broken_redirect_list.js:1077]: @internetisaiah this redirect is not working for me 🤔

Comment (bre-fitzgerald) [reply assets/js/broken_redirect_list.js:1077]: The trailing slashes might be required

File: _docs/_partners/data_and_infrastructure_agility.md [hunk 2]
```diff
@@ -107,8 +107,8 @@ valid_partner_list:
   url: /docs/partners/data_and_infrastructure_agility/customer_data_platform/actioniq/
 - name: Airbyte
   url: /docs/partners/data_and_infrastructure_agility/workflow_automation/airbyte/
-- name: Flywheel
-  url: /docs/partners/data_and_infrastructure_agility/workflow_automation/flywheel/
+- name: GrowthLoop
```
Comment (bre-fitzgerald) [inline _docs/_partners/data_and_infrastructure_agility.md:110]: @internetisaiah you've got 2 out of 3 tile updates! The tile also needs to be updated on the [Workflow Automation](https://www.braze.com/docs/partners/data_and_infrastructure_agility/workflow_automation) parent page

**Comment by lauren-stein** [comment]
No worries! You mentioned you are working with my team to get an image. Do
I need to send you something?

File: assets/js/broken_redirect_list.js (hunk 1 above)
Comment (internetisaiah) [reply assets/js/broken_redirect_list.js]: _Note: escalated in Slack. The issue is probably related to [this PR](https://github.com/braze-inc/braze-docs/pull/6636/files) and is not a bug/issue introduced in this branch_.

**Comment by internetisaiah** [comment]
@lauren-stein ,

Nope, we should be good now! We were able to get the image! 🙌🏽 

A member of my team is currently reviewing this PR and then once approved I'll merge your changes and then these updates will be publicly visible on the docs site after our deployment tomorrow (or Tuesday).

Thanks for all your help with this!

**Review by bre-fitzgerald — APPROVED** [review]

**Comment by lauren-stein** [comment]
Wonderful, that sounds great. Thank you again!

File: assets/js/broken_redirect_list.js (hunk 1 above)
Comment (internetisaiah) [reply assets/js/broken_redirect_list.js]: _Note: I pulled in changes from `origin/develop` and redirects are working properly in my local env._

**Comment by internetisaiah** [comment]
@lauren-stein ,

Changes have been merged into the `develop` branch ✅ 

They should be pushed to the live site (i.e. our `master` branch) tomorrow during our weekly Thursday deploy! After that, your changes should be reflected in the docs within ~24 hours.

If you catch any issues or places that we missed for the rebrand, let me know and I'll help get it fixed right away 👍🏽

**Comment by lauren-stein** [comment]
Thank you so much!

**Pull request merged**